lib/pwwka/configuration.rb

```diff
@@ -32,6 +33,7 @@ def initialize
       @async_job_klass = Pwwka::SendMessageAsyncJob
       @default_prefetch = nil
       @receive_raw_payload = false
+      @process_name = ""
```

Comment:
Would it make sense to default this to `$0`?  I'm not sure what the value that would be in our infra, but it might be a good default?

Reply:
Yeah, I can do that.  What is a typical value for `$0` in standard infrastructures?

Reply:
I'm…not sure actually.  I think it highly depends on how the thing is run.  For us it might be `dumb-init` since that is what's in docker.

Reply:
I do have some concerns with this security wise. Although it wouldn't affect us my main concern is that its exposing some data that downstream users are unaware of/not expecting. I think it may be better to default this to nothing instead of the name sent.

Reply:
This is what showed up in my test running in staging 
`c: RailsRabbitTest /usr/local/bundle/bin/rake`